src/utils/source-search.js

```diff
@@ -200,9 +210,10 @@ function clearSearch(cm) {
  * @memberof utils/source-search
  * @static
  */
-function find(ctx, query, keepSelection) {
+function find(ctx, query, caseSensitive, wholeWord, regexMatch) {
```

Comment:
👍 I had kind of thought so too after looking over all the changes 😆 I'll do that, pass in a modifiers object.